spec/rubocop/cop/rspec/empty_example_group_spec.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::RSpec::EmptyExampleGroup, :config do
-  it 'flags an empty context' do
-    expect_offense(<<-RUBY)
+  it 'flags an empty example group' do
+    expect_offense(<<~RUBY)
```

Comment:
Unrelated change, I can roll this back and fix it for all in a separate pull request if you like.
The failure spec output looks slightly better with `<<~`.